cabal-install/tests/UnitTests/Distribution/Client/IndexUtils.hs

```diff
@@ -0,0 +1,79 @@
+module UnitTests.Distribution.Client.IndexUtils where
```

Comment:
Added these simple unit-tests for `preferred-versions` parser, since I thought, they should have some tests.

Therefore, we have to expose a bit more from `IndexUtils` to be able to test it.